runtime/Cpp/runtime/src/tree/xpath/XPathLexer.cpp

```diff
@@ -46,14 +47,14 @@ const atn::ATN& XPathLexer::getATN() const {
 
 void XPathLexer::action(RuleContext *context, size_t ruleIndex, size_t actionIndex) {
   switch (ruleIndex) {
-    case 4: IDAction(dynamic_cast<RuleContext *>(context), actionIndex); break;
+    case 4: IDAction(dynamic_cast<antlr4::RuleContext *>(context), actionIndex); break;
 
   default:
     break;
   }
 }
 
-void XPathLexer::IDAction(RuleContext * /*context*/, size_t actionIndex) {
+void XPathLexer::IDAction(antlr4::RuleContext *context, size_t actionIndex) {
```

Comment:
Are these changes manual or generated? The outcommented context name is necessary to silence a compiler warning (unused parameter), so such things shouldn't be changed lightly.